openshift/datagrid-service/prometheus-monitoring/README.md

```diff
@@ -0,0 +1,156 @@
+Monitoring Red Hat Data Grid (RHDG) for OpenShift
+=================================================
+**Author:** Vladimir Blagojevic, Don Naro  
+**Technologies:** Infinispan, Data Grid, OpenShift  
+**Summary:** Learn how to integrate RHDG/Prometheus/Grafana on Openshift platform  
+**Target Product:** Red Hat Data Grid  
+**Product Versions:** RHDG 7.3 or later
+
+About This Quickstart
+---------------------
+This quickstart demonstrates how you can monitor Data Grid for OpenShift with Prometheus and Grafana.
+
+**NOTE:** Data Grid for OpenShift supports Prometheus monitoring with both `cache-service` and `datagrid-service`. However, this quickstart is designed to work with a `datagrid-service` pod in a specific namespace.
```

Comment:
If the tutorial applies to either cache or datagrid service, I think it should be located in `openshift/shared`. It's fine to just use one of the services and note it here.